decidim-core/app/packs/src/decidim/redesigned_heading_tag.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Ensures the aside heading tag is transformed to h2 or h1 if the drawer is shown or hidden
+ * @param {HTMLElement} node target node
+ * @returns {void}
+ */
+export default function(node = document) {
```

Comment:
Do we still need this after the discussion about handling headings?

Reply:
Because we are removing drawers in another issue, for the moment we are merging this code.